python/test/unit/fem/test_petsc_solver_wrappers.py

```diff
@@ -18,9 +18,11 @@
 @pytest.mark.petsc4py
 class TestPETScSolverWrappers:
     @pytest.mark.parametrize(
-        "mode", [dolfinx.mesh.GhostMode.none, dolfinx.mesh.GhostMode.shared_facet]
+        "mode",
+        [dolfinx.mesh.GhostMode.none, dolfinx.mesh.GhostMode.shared_facet],
     )
-    def test_compare_solvers(self, mode):
+    @pytest.mark.parametrize("nonlinear_solver", ["snes high level", "legacy Newton"])
+    def test_compare_solvers(self, mode, nonlinear_solver):
```

Comment:
Test is comparing solutions, not solvers.

Reply:
I gave it another try (name change)